data-access-layer/db/DBManager/DBManager.js

```diff
@@ -118,26 +117,7 @@ class DBManager {
         minDelay: 1000
       })
         .catch(Timeout, throwTimeoutError)
-        .then(resource => utils.decodeBase64(_.get(resource, 'status.response')))
-        .catch(NotFound, () => {
-          // TODO: This can be removed from T2018.13B onwards as bind property would have been moved to ApiServer by then.
-          return utils
-            .retry(() => this
-              .directorService
-              .getBindingProperty(config.mongodb.deployment_name, CONST.FABRIK_INTERNAL_MONGO_DB.BINDING_ID), {
-              maxAttempts: 5,
-              minDelay: 5000,
-              predicate: err => !(err instanceof ServiceBindingNotFound)
-            })
-            .then(bindInfo => {
-              return this.storeBindPropertyOnApiServer({
-                id: _.toLower(CONST.FABRIK_INTERNAL_MONGO_DB.BINDING_ID),
-                parameters: bindInfo.parameters,
-                credentials: bindInfo.credentials
-              })
-                .then(() => bindInfo);
-            });
-        });
+        .then(resource => utils.decodeBase64(_.get(resource, 'status.response')));
```

Comment:
Please validate that the corresponding apiserver resource is present in higher landscapes, also in new landscapes.

Reply:
1. Test the re-creation with deleting sf-event-mesh-server and sf deployment and re-creating them.

Reply:
Both done.